dockstore-client/src/main/java/io/dockstore/client/cli/nested/CwltoolLauncher.java

```diff
@@ -47,11 +47,13 @@ public List<String> buildRunCommand() {
 
         extraFlags = extraFlags.stream().map(string -> string.split(",")).flatMap(Arrays::stream).map(this::trimAndPrintInput)
                 .collect(Collectors.toList());
-
         // Create base execution command
         CWLRunnerInterface cwlRunner = CWLRunnerFactory.createCWLRunner();
         command = cwlRunner.getExecutionCommand(workingDirectory + "/outputs/", workingDirectory + "/tmp/", workingDirectory + "/working/",
-                primaryDescriptor.getAbsolutePath(), provisionedParameterFile.getAbsolutePath());
+                primaryDescriptor.getAbsolutePath(), java.util.Optional.empty());
```

Comment:
Nit, unless there's a conflict, I'd make this `Optional.empty()` and do an import